crates/gosub_shared/src/lib.rs

```diff
@@ -2,6 +2,9 @@
 //!
 //! This crate supplies a lot of shared functionality in the gosub engine.
 //!
+
+extern crate core;
+
```

Comment:
Why is this needed? Shouldn't you be able to just use the `core` crate? If not, it also should be available through the std crate

Reply:
it's an automatic thing that goland seems to add. It also does not do a very good check, as sometimes it adds it twice.  I can remove this.